src/components/editor/Editor.jsx

```diff
@@ -82,12 +82,12 @@ Editor.propTypes = {
   location: PropTypes.object,
   resourceTemplateId: PropTypes.string,
   history: PropTypes.object,
-  authenticationState: PropTypes.object
+  currentSession: PropTypes.object
 }
 
 const mapStateToProps = (state) => {
   return {
-    authenticationState: Object.assign({}, state.authenticate.authenticationState)
+    currentSession: Object.assign({}, state.authenticate.authenticationState.currentSession)
```

Comment:
We've been gradually moving towards using the spread operator for copying state from array- and object-like things where posible, so I think this could be reworked as:

```javascript
currentSession: { ...state.authenticate.authenticationState.currentSession }
```